tests/Unit/Emogrifier/Support/Tests/AssertCssTest.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+namespace Pelago\Tests\Unit\Emogrifier\Support\Tests;
+
+use Pelago\Tests\Unit\Emogrifier\Support\Tests\Fixtures\TestingAssertCss;
+
+/**
+ * Test case.
+ */
```

Comment:
If you like, you can add yourself as an `@author` of the new files so proper credit is given.